### PR TITLE
Offline worst case trade fees

### PR DESCRIPTION
**Pull request by gloriousCode**
# Description

Adds offline trade fee calculation for each exchange using worst case values. On the wrapper, if the keys aren't set, then switch the fee type to offline to prevent API calls.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

I've added a logic test `TestGetFeeByTypeOfflineTradeFee` to check if offline trade fee feetype gets hit

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Travis with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/exchange.go [hunk 1]
```diff
@@ -31,41 +31,42 @@ const (
...
 
 // Const declarations for fee types
 const (
-	BankFee                        FeeType = "bankFee"
-	InternationalBankDepositFee    FeeType = "internationalBankDepositFee"
-	InternationalBankWithdrawalFee FeeType = "internationalBankWithdrawalFee"
-	CryptocurrencyTradeFee         FeeType = "cryptocurrencyTradeFee"
-	CyptocurrencyDepositFee        FeeType = "cyptocurrencyDepositFee"
-	CryptocurrencyWithdrawalFee    FeeType = "cryptocurrencyWithdrawalFee"
+	BankFee                        FeeType = 0
```
Comment (thrasher-) [inline exchanges/exchange.go]: This can be simplified to iota and the values below will automatically be incremented

File: exchanges/exchange.go [hunk 2]
```diff
@@ -31,41 +31,42 @@ const (
...
-	RapidTransfer   InternationalBankTransactionType = "rapidTransfer"
-	MisterTangoSEPA InternationalBankTransactionType = "misterTangoSepa"
-	Qiwi            InternationalBankTransactionType = "qiwi"
-	VisaMastercard  InternationalBankTransactionType = "visaMastercard"
-	WebMoney        InternationalBankTransactionType = "webMoney"
-	Capitalist      InternationalBankTransactionType = "capitalist"
-	WesternUnion    InternationalBankTransactionType = "westernUnion"
-	MoneyGram       InternationalBankTransactionType = "moneyGram"
-	Contact         InternationalBankTransactionType = "contact"
+	WireTransfer    InternationalBankTransactionType = 0
```
Comment (thrasher-) [inline exchanges/exchange.go]: Same here

File: exchanges/okex/okex_test.go [hunk 3]
```diff
@@ -1551,26 +1551,6 @@ func TestGetETTOrderDetails(t *testing.T) {
 	testStandardErrorHandling(t, err)
 }
 
-// TestGetETTConstituents API endpoint test
```
Comment (thrasher-) [inline exchanges/okex/okex_test.go]: Can t.Skip these, if they don't bring them back online then we can remove the functions and associated tests.

Comment (gloriousCode) [reply exchanges/okex/okex_test.go]: Far out that wasn't thought through 😄 "didn't work, better remove the tests!" Will add back. Can remove ETT if it doesn't return in another PR later 😄

**Pull request merged**